subsys/mgmt/mcumgr/lib/cmd/img_mgmt/src/img_mgmt_state.c

```diff
@@ -6,6 +6,7 @@
 
 #include <assert.h>
 #include <zephyr/sys/util_macro.h>
+#include <zephyr/toolchain/common.h>
```

Comment:
`zephyr/toolchain/common.h` is more of an "internal" header; usually, the recommend way to include this header would be through `zephyr/toolchain.h`.

Reply:
Thanks. Switched this and the one in img_mgmt.c as mentioned by @gmarull